docker/1.4.0/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,111 @@
+    build-essential \
+    ca-certificates \
+    cmake \
+    curl \
+    git \
+    jq \
+    libglib2.0-0 \
+    libgl1-mesa-glx \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    vim \
+    wget \
+    zlib1g-dev \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && gunzip -c openmpi-$OPEN_MPI_VERSION.tar.gz | tar xf - \
+ && cd openmpi-$OPEN_MPI_VERSION \
+ && ./configure --prefix=/home/.openmpi \
+ && make all install \
+ && cd .. \
+ && rm openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && rm -rf openmpi-$OPEN_MPI_VERSION
+
+# The ENV variables declared below are changed in the previous section
+# Grouping these ENV variables in the first section causes
+# ompi_info to fail. This is only observed in CPU containers
+ENV PATH="$PATH:/home/.openmpi/bin"
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
+RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value
+
+RUN curl -o ~/miniconda.sh -O  https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+ && chmod +x ~/miniconda.sh \
+ && ~/miniconda.sh -b -p /opt/conda \
+ && rm ~/miniconda.sh \
+ && /opt/conda/bin/conda install -y -c anaconda \
+    python=$PYTHON_VERSION \
+    numpy==1.16.4 \
+    ipython==5.8.0 \
+    mkl==2019.4 \
+    mkl-include==2019.4 \
+    cython==0.29.12 \
+    typing==3.7.4 && \
+    /opt/conda/bin/conda clean -ya
+
+RUN conda install -c conda-forge \
+    awscli==1.17.7 \
+    opencv==4.0.1 \
+ && conda install -y \
+    scikit-learn==0.20.3 \
+    pandas==0.24.2 \
+    Pillow==6.2.0 \
+    h5py==2.9.0 \
+    requests==2.22.0 \
+ && conda clean -ya \
+ && pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org
+
+# Copy workaround script for incorrect hostname
+COPY changehostname.c /
+COPY start_with_right_hostname.sh /usr/local/bin/start_with_right_hostname.sh
+COPY deep_learning_container.py /usr/local/bin/deep_learning_container.py
+COPY sagemaker_pytorch_container-1.4.0-py2.py3-none-any.whl /sagemaker_pytorch_container-1.4.0-py2.py3-none-any.whl
+
+# The following section uninstalls torch and torchvision before installing the
+# custom versions from an S3 bucket. This will need to be removed in the future
+RUN pip install --no-cache-dir \
+    scipy==1.2.2 \
+    future \
+    /sagemaker_pytorch_container-1.4.0-py2.py3-none-any.whl \
+ && pip uninstall -y torch \
+ && pip install -U --no-cache-dir https://pytorch-aws.s3.amazonaws.com/pytorch-1.4.0/py2/cpu/torch-1.4.0-cp27-cp27mu-manylinux1_x86_64.whl \
+ && pip uninstall -y torchvision \
+ && pip install --no-cache-dir \
+    https://torchvision-build.s3.amazonaws.com/1.4.0/py2/cpu/torchvision-0.5.0%2Bcpu-cp27-cp27mu-linux_x86_64.whl \
+ && rm /sagemaker_pytorch_container-1.4.0-py2.py3-none-any.whl
+
+WORKDIR /
+
+RUN chmod +x /usr/local/bin/start_with_right_hostname.sh \
+ && chmod +x /usr/local/bin/deep_learning_container.py
```

Comment:
Note: we have removed this file, please remove references to it (see https://github.com/aws/sagemaker-pytorch-container/pull/156 for how to make the changes)